hyp3lib/rtc2color.py

```diff
@@ -120,6 +153,7 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   else:
     green = (3.0*np.sqrt(xp)*(1 - blue_mask) + 2.0*zp*blue_mask)*255
   green[green==0]=1
+  # FIXME: Is this the right mask (xp > 0)? Green should be both copol and crosspol
   green = green * mask
```

Comment:
>     green: volume (depolarized) scattering

This is attempting to mask our all no data and cleaned values, but only uses crosspol to define that. Green should be both copol and crosspol. Does it make sense to mask based on crosspol only?

Reply:
This is the right mask as long as the cleanup threshold is << than the user selected power threshold.